dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/db/MsSql.kt

```diff
@@ -17,9 +17,7 @@ public object MsSql : DbType("sqlserver") {
     override val driverClassName: String
         get() = "com.microsoft.sqlserver.jdbc.SQLServerDriver"
 
-    override fun convertSqlTypeToColumnSchemaValue(tableColumnMetadata: TableColumnMetadata): ColumnSchema? {
-        return null
-    }
+    override fun convertSqlTypeToColumnSchemaValue(tableColumnMetadata: TableColumnMetadata): ColumnSchema? = null
```

Comment:
is it really linter setting? could we change it? I don't like these short forms without brackets, it looks ugly and make me slower

Reply:
It is https://pinterest.github.io/ktlint/latest/rules/standard/#function-expression-body.
Because of the official style guide: https://kotlinlang.org/docs/coding-conventions.html#functions.
To me, block bodies with just one return statement take up unnecessary space and feel like java-boilerplate. We could change it, it's a matter of taste. But I think the most important thing is consistency; Mixing single-line block body's and expression bodies is messy.